src/main_02008254.c

```diff
@@ -0,0 +1,18 @@
+#include "main_02008254.h"
+
+s32 sub_0207F818(struct file_stream* file, u8* buf, s32 size);
+void CardPullOutWithStatus(u32 status);
```

Comment:
note to self: probably a good place to start from in the next cr, though need to figure out the bx ip stuff before long as well